docs/integration-guide.md

```
@@ -20,18 +20,21 @@ const GROUP_SERVICE_DID = 'did:web:dev.groups.certified.app'
 
 ## Architecture: where your app fits
 
+The current reference integration uses **direct calls to the configured CGS URL**:
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)